### DISCUSSION TITLE
Бриф

### DISCUSSION THREAD

**Discussion by Darolga** [torin-asakura/drum-in#7] [General]
**Цели Руководителя**

1. Для чего вам нужен сайт? Какие приоритетные задачи должен решать? 
(Т.к. разрабатываем два сайта - опишите цели и задачи каждого)

а) Онлайн курс

б) Индивидульные занятия

2. Какое место в вашем бизнесе отведено каждому сайту? 

4. Проект новый или это доработка старого?

5. Какие финансовые выгоды вы ожидаете получить от Онлайнкурса и от сайта по Индивидуальным занятиям?

6. Как сайты могут повлиять на ваш бизнес в целом?

7. Какие видите перспективы развития сайтов в вашем бизнесе?

7. Есть ли маркетинговый план? На какой уровень хотите выйти и каких целей достичь?

**Цели и задачи сайта**

1. На чем зарабатывает каждый из сайтов?
2. Как сайт влияет/ должен повлиять/ на ваш основной вид деятельности?
3. Опишите проблемы существующего сайта, которые требуют решения.

**Пользователь сайта Онлайн курса**

1. Кто ваша целевая аудитория? Возраст, пол, социальное положение, интересы.
2. Если это несколько сегментов, опишите каждый - кто они и с какой цели приходит на сайт.

а) 
б)
в)

3. Опишите идеальный сценарий поведения пользователя (с точки зрения целей бизнеса) .

4. Как осуществляется поддержка (взаимодействие, обратная связь) с пользователем?

5. Как осуществляется техническая поддержка сайта? (админы...)

6. Кто помимо пользователей будет использовать систему?
7. Ведется ли поддержка соцсетей? Как часто и какие материалы выкладываются там (хотели бы выкладывать)
8. Как хотите упростить жизнь себе и пользователям?

**Пользователь сайта по Индивидульным занятиям.**

1. Кто целевая аудитория? Возраст, пол, социальное положение, интересы.
2. Если это несколько сегментов, опишите каждый - кто они и с какой цели приходит на сайт.

а) 
б)
в)

3. Опишите идеальный сценарий поведения пользователя (с точки зрения целей бизнеса)
4. Как осуществляется поддержка (взаимодействие, обратная связь) с пользователем?
5. Как осуществляется техническая поддержка сайта? 
6. Кто помимо пользователей будет использовать систему?
7. Ведется ли поддержка соцсетей? Как часто и какие материалы выкладываются там (хотели бы выкладывать)
8. Как хотите упростить жизнь себе и пользователям?

**Бренд**

1. Есть ли лого и фирстиль?
2. Делаем один лого для обоих сайтов или два разных?
3. Если нужен новый лого, каким он должен быть?

а) Примеры тех, кто нравится
б) Примеры тех, кто не нравится.

3. Есть ли узнаваемость бренда/личности сейчас? 
4. Что хотели бы изменить, добавить?
5. Пожелания (стиль, цвет, на чем сделать акценты и т.д.)

**Конкуренты**

1. Кто ваши конкуренты?
2. Есть ли у конкурентов сайт и какую роль он выполняет?
3. На кого вы ориентируетесь как пример?
4. Покажите решения конкурентов, которые вам нравятся именно вам и почему?

а) Примеры тех, кто нравится и почему.
б) Примеры тех, кто не нравится и почему.

5. Чем вы лучше других? (фичи, в чем уникальность). В чем ваши сильные стороны?
6. Какие преимущества будут у вашего сайта как продукта по сравнению с конкурентами ?
6. Какие барьеры? Что мешает, чего не хватает?
7. Как хотите масштабироваться?

**Comment by MarinaDrumin** [comment]
> **Цели Руководителя**
> 
> 
> 
> 1. Для чего вам нужен сайт? Какие приоритетные задачи должен решать? 
> 
> (Т.к. разрабатываем два сайта - опишите цели и задачи каждого)
> 
> 
> 
> а) Онлайн курс
> размещение и продажа курсов,информация о преподавателе
> 
> 
> б) Индивидульные занятия
> информация о преподавателе, продажа индивидуальных занятий 
> 
> 
> 2. Какое место в вашем бизнесе отведено каждому сайту? 
> 
> 1- 60%, 2- 40%
> 
> 4. Проект новый или это доработка старого?
> 1. Новый
2. Доработка старого
> 
> 
> 5. Какие финансовые выгоды вы ожидаете получить от Онлайнкурса и от сайта по Индивидуальным занятиям?
> 1. 150000 в  мес
> 2. 30000 в мес
> 
> 6. Как сайты могут повлиять на ваш бизнес в целом?
> 1. Масштабирование
> 2. Поддержание потока клиентов
> 
> 7. Какие видите перспективы развития сайтов в вашем бизнесе?
> 
> 1. Увеличение количества курсов
> 
> 7. Есть ли маркетинговый план? На какой уровень хотите выйти и каких целей достичь?
> 
> см п.5
> 
> 
> 
> **Цели и задачи сайта**
> 
> 
> 
> 1. На чем зарабатывает каждый из сайтов?
> 1. Продажа курса
2. Индивидуальнее занятия
> 2. Как сайт влияет/ должен повлиять/ на ваш основной вид деятельности?
> 1. Создание и проодажа новых программ
> 3. Опишите проблемы существующего сайта, которые требуют решения.
> 1) Лаконичность
2) простой дизайн
3) текст сайта возможно? не адаптирован под поиск
> 
> 
> 
> 
> **Пользователь сайта Онлайн курса**
> 
> 
> 
> 1. Кто ваша целевая аудитория? Возраст, пол, социальное положение, интересы.
>  1.музыканты, играющий на любом инструменте более года, 
Вокалист, композитор, мужчины и женщины от 25 до 35, студенты последних курсов, работающие
2. Дети от 10 лет,взрослые от 18 до 35 лет, м и ж, 
Интересы - музыка, музыкальные инструменты, написание песен, пение, концерты, фестивали

> 2. Если это несколько сегментов, опишите каждый - кто они и с какой цели приходит на сайт.
> 
> 
> 1)
> а) музыканты, профессиональный рост
> 
> б)композиторы, профессиональный рост
> 
> в)певцы, профессиональный рост 
> 
> 2) а)дети- развитие, досуг, музыкальное образование
> б) взрослые - досуг, поднятие настроения, самооценки

> 3. Опишите идеальный сценарий поведения пользователя (с точки зрения целей бизнеса) .
> 1.ознакомился с сайтом, оставил заявку, внёс оплату за Уровень
> 2. Заинтересовался экспертом, посмотрел место занятий, связался и договорился о занятии
> 
> 4. Как осуществляется поддержка (взаимодействие, обратная связь) с пользователем?
> 1)телеграмм, Вотсап, 
2) телефон, тг, Вотсап
> 
> 
> 5. Как осуществляется техническая поддержка сайта? (админы...)
> 
> ?
> 
> 6. Кто помимо пользователей будет использовать систему?
> маркетолог
> 7. Ведется ли поддержка соцсетей? Как часто и какие материалы выкладываются там (хотели бы выкладывать)
> вк- раз в неделю,
Фб - 2 раза в неделю
Инста - 2 раза в неделю 

> 8. Как хотите упростить жизнь себе и пользователям?
> 
> 1. Давать инфу о наборе раз в месяц/два
> Возможность регистрации и открытия доступа курсу по мере оплаты 
> 2. Быстрая связь 
> 
> **Пользователь сайта по Индивидульным занятиям.**
> 
> 
> 
> 1. Кто целевая аудитория? Возраст, пол, социальное положение, интересы.
> см выше п.2
> 2. Если это несколько сегментов, опишите каждый - кто они и с какой цели приходит на сайт.
> 

 см выше п.2

> 
> а) 
> 
> б)
> 
> в)
> 
> 
> 
> 
> 
> 3. Опишите идеальный сценарий поведения пользователя (с точки зрения целей бизнеса)
>  см выше п.2
> 4. Как осуществляется поддержка (взаимодействие, обратная связь) с пользователем?
>  см выше п.2
> 5. Как осуществляется техническая поддержка сайта? 
> ?
> 6. Кто помимо пользователей будет использовать систему?
>  см выше 
> 7. Ведется ли поддержка соцсетей? Как часто и какие материалы выкладываются там (хотели бы выкладывать)
>  см выше 
> 8. Как хотите упростить жизнь себе и пользователям?
> 
>  см выше п.2
> 
> **Бренд**
> 
> 
> 
> 1. Есть ли лого и фирстиль? Есть Лого и стиль
> 
> 2. Делаем один лого для обоих сайтов или два разных? Лого один, стили разные, разное цветовое решение 
> 
> 3. Если нужен новый лого, каким он должен быть?
> 
> 
> 
> а) Примеры тех, кто нравится
> 
> б) Примеры тех, кто не нравится.
> 
> 
> 
> 3. Есть ли узнаваемость бренда/личности сейчас? 
> 
> 4. Что хотели бы изменить, добавить?
> 
> 5. Пожелания (стиль, цвет, на чем сделать акценты и т.д.)
> 
> 1. Голубой, фиолетовый

> 2. Можно оставить красно-бело-чёрный 
> 
> 
> **Конкуренты**
> 
> 
> 
> 1. Кто ваши конкуренты?
> 1. Школа Ритма Льва Cлепнера, 
2. Нет
> 2. Есть ли у конкурентов сайт и какую роль он выполняет?
> https://shkola-ritma.ru/
> 3. На кого вы ориентируетесь как пример?
> на https://stereoschool.ru/

https://drumschool.ru/

> 4. Покажите решения конкурентов, которые вам нравятся именно вам и почему?
> https://drumschool.ru/
Стильные фото, качественные контент
> https://stereoschool.ru/
> множество программ 
Удобно выбирать
> а) Примеры тех, кто нравится и почему.
> https://college.berklee.edu/
Дизайн, цвета, структура
> б) Примеры тех, кто не нравится и почему.
> 
> https://shkola-ritma.ru/
Недоооформленность, присутствует хаотичность структуры
> 
> 5. Чем вы лучше других? (фичи, в чем уникальность). В чем ваши сильные стороны?
> более 6 мировых школ Ритма, уникальные разработки, 9 летний опыт, более 500 учившихся 
> 6. Какие преимущества будут у вашего сайта как продукта по сравнению с конкурентами ?
> доступ к курсу
> 6. Какие барьеры? Что мешает, чего не хватает?
> времени, знаний в таргете 
> 7. Как хотите масштабироваться?
> 
> реклама
> 
> 
> 
>

**Comment by Darolga** [comment]
**Вопросы по контенту Онлайн курса,** 
к документу https://docs.google.com/document/d/1-MGgw7iroEV6vzG71Aeh_ppH5tNMQ1kkrwQ1TEeTKEw/edit# 

1. Онлайн Курс состоит из 3х частей. 
Часть 1.  Ощущение Времени - TIME FILLING
Часть 2.  Пятое Измерение - FIVE  MEASURE
ЧАСТЬ 3. Седьмое Небо- SEVEN SKY

Правильно ли понимаю, что каждую часть можно оплатить отдельно и проходить только ее? 

Из документа:
ОПЛАТА КУРСА
В месяц - 7400р.
Каждая следующая часть - со скидкой 10%

2. В документе далее описаны еще два раздела:
2) КОНОКОЛ
3) КЛЮЧИ ПОЛИРИТМИИ

Не совсем понятно, это два отдельных онлайнкурса или  все части одного ОНЛАЙН-КУРСа  «ОТКРЫТИЕ РИТМА»?

3. Плинируете в будущем увеличивать кол-во онлайн-курсов и программ, продаваемых на сайте, верно?

4. Если есть отзывы прошедших онлайн-курс, будет полезно показать на сайте.

**Вопросы по контенту Индивидуальные занятия** 

1. Не нашла информации на сайте https://druminstructor.ru/ о ценах на индивидуальные занятия. Стоит указать.
 Если онлайн-занятия стоят дешевле, то об этом тоже нужно сказать. 



**Предложения обоим сайтам:**

1. Не совсем понятна Целевая аудтитория для каждого сайта.
 Дети (от 10 лет) указаны как целевая аудитория - относятся к индивидуальным занятиям? Или смогут осилить Онлайнкурс?

2. Для новичков (и для родителей подростков) полезно было бы дать ссылки, где можно приобрести инструмент для обучения (палок, инструмента или тренировочного пэда). Если их можно приобрести у вас, то нужно сказать об этом. 

2. Есть ли фото и видео-материалы с выступлений преподавателя? Было бы полезно видео, демонстрирующего мастерство преподавателя для онлайн-курса. А также видео о том, как проходит индивидуальный урок очно у учеников разной степени подготовки для Индивидальных занятий.

Если есть, дайте, пожалуйста, исходники в большом разрешении ( или ссылки) тех, которые можно разместить на сайте. 

4. Было бы круто записать и разместить на сайтах небольшое видео-обращение преподавателя.

**Reply by MarinaDrumin** [reply]
> **Вопросы по контенту Онлайн курса,** к документу https://docs.google.com/document/d/1-MGgw7iroEV6vzG71Aeh_ppH5tNMQ1kkrwQ1TEeTKEw/edit#
> 
> 1. Онлайн Курс состоит из 3х частей.
>    Часть 1.  Ощущение Времени - TIME FILLING
>    Часть 2.  Пятое Измерение - FIVE  MEASURE
>    ЧАСТЬ 3. Седьмое Небо- SEVEN SKY
> 
> Правильно ли понимаю, что каждую часть можно оплатить отдельно и проходить только ее?
> Да, я заменила слово Часть на Уровень. Можно проходить отдельно, но для прохождения 2 и 3 уровня нужно знать основы. 
> Из документа: ОПЛАТА КУРСА В месяц - 7400р. Каждая следующая часть - со скидкой 10%
> 
> 2. В документе далее описаны еще два раздела:
> 
> 2. КОНОКОЛ
> 3. КЛЮЧИ ПОЛИРИТМИИ
> 
> Не совсем понятно, это два отдельных онлайнкурса или все части одного ОНЛАЙН-КУРСа «ОТКРЫТИЕ РИТМА»?
> это два отдельных курса, которые в планах
> 3. Плинируете в будущем увеличивать кол-во онлайн-курсов и программ, продаваемых на сайте, верно?
+
> 4. Если есть отзывы прошедших онлайн-курс, будет полезно показать на сайте.
> есть видео отзывы о курсе и о индивидуальных занятиях - добавлю ссылки в документ
> **Вопросы по контенту Индивидуальные занятия**
> 
> 1. Не нашла информации на сайте https://druminstructor.ru/ о ценах на индивидуальные занятия. Стоит указать. 
Цены меняются - 
на данный момент индивидуальное занятие по барабанам очно и онлайн стоит 1500р
Индивидуальное Онлайн-занятие по курсу - 2000р
>    Если онлайн-заятия стоят дешевле, то об этом тоже нужно сказать.
> 
> **Предложения обоим сайтам:**
> 
> 1. Не совсем понятна Целевая аудтитория для каждого сайта.
>    Дети (от 10 лет) указаны как целевая аудитория - относятся к индивидуальным занятиям? Или смогут осилить Онлайнкурс?
Дети - только очные занятия по барабанам
> 2. Для новичков (и для родителей подростков) полезно было бы дать ссылки, где можно приобрести инструмент для обучения (палок, инструмента или тренировочного пэда). Если их можно приобрести у вас, то нужно сказать об этом.
у меня - нельзя приобрести, рекомендации и ссылки выдаю по запросу
> 3. Есть ли фото и видео-материалы с выступлений преподавателя?
Все ссылки на видео выложены в разделе "о преподавателе"

Было бы круто записать и разместить на сайтах небольшое видео-обращение преподавателя.

Да, работаю над этим!

**Comment by Darolga** [comment]
**Есть ли фото и видео-материалы с выступлений преподавателя?**
Все ссылки на видео выложены в разделе "о преподавателе"

На сайте фото небольшие. Можете дать исходники? 

Вот здесь тоже есть красивые фото https://vk.com/albums-9435826?z=photo-9435826_457239125%2Fphotos-9435826
По цвету подошли бы. Можно их использовать?

**Reply by MarinaDrumin** [reply]
> **Есть ли фото и видео-материалы с выступлений преподавателя?** Все ссылки на видео выложены в разделе "о преподавателе"
> 
> На сайте фото небольшие. Можете дать исходники?
> 
> Вот здесь тоже есть красивые фото https://vk.com/albums-9435826?z=photo-9435826_457239125%2Fphotos-9435826 По цвету подошли бы. Можно их использовать?

конечно
вы не нашли ссылки на выступления в Гуглдоке?
я там добавила также отзывы

**Comment by MarinaDrumin** [comment]
плакаты cо старым лого https://disk.yandex.ru/d/hhVVOdFwpcMkEA

фото для сайта https://disk.yandex.ru/d/jUp6RDBlx2kjqQ

**Comment by MarinaDrumin** [comment]
[гуглдок
https://docs.google.com/document/d/1-MGgw7iroEV6vzG71Aeh_ppH5tNMQ1kkrwQ1TEeTKEw/edit?usp=sharing
